subsys/bluetooth/mesh/net.c

```diff
@@ -257,24 +257,19 @@ bool bt_mesh_net_iv_update(uint32_t iv_index, bool iv_update)
 			return false;
 		}
 
-		if (iv_index > bt_mesh.iv_index + 1) {
+		if ((iv_index > bt_mesh.iv_index + 1) ||
+		    (iv_index == bt_mesh.iv_index + 1 && !iv_update)) {
```

Comment:
this condition requires to be 'and' with 96 hours rule checking

Reply:
iv_upate is only false here if "missed the Secure Network beacons with the IV update Flag set to 1". Then this have to wait 96 hours before it will be updated, and can come back into the network. That is why the 96 hours rule is left out.

Reply:
>this condition requires to be 'and' with 96 hours rule checking 

No, this is precisely what we discussed and we don't want. This situation does not arise if the node is continuously operating in a network. This situation arises: when node is powered off for many hours or days before coming alive back; meanwhile, rest of the network has moved to the next IV index (old iv + 1, and in Normal operation).